llvm/tools/llvm-profgen/ProfileGenerator.cpp

```diff
@@ -64,6 +64,12 @@ static cl::opt<bool>
                              "than threshold, it will be trimmed."),
                     cl::cat(ProfGenCategory));
 
+static cl::opt<bool> ForceProfilePreinlined(
+    "force-profile-preinlined",
```

Comment:
nit: `mark-all-context-preinlined`? we are not force doing actual reinline, but only marking preinlined flag.